packages/lambda-tiler/src/routes/health.ts

```diff
@@ -87,6 +86,7 @@ export async function Health(req: LambdaHttpRequest): Promise<LambdaHttpResponse
 
         // Get test tile to compare
         const testBuffer = await getTestBuffer(test);
+        test.buf = testBuffer;
```

Comment:
Is this `test.buf` been used by anything?

Reply:
I believe it is somewhere, I didnt want to break backwards compatibility